app/assets/javascripts/tags.js.coffee

```diff
@@ -1,17 +1,34 @@
-$(document).on "click", ".switch_offer", (event) ->
-  $.ajax '/tags/offers',
-    type: 'GET'
-    dataType: 'html'
-    error: (jqXHR, textStatus, errorThrown) ->
-        $('.alpha_tag_list').html "AJAX Error: #{textStatus}"
+$ ->
+  $(".switch_offer-js").on "click", ->
```

Comment:
Turbolinks? PJAX? All these are programmatical. And I intend to have Turbolinks 3 in place as soon as we can touch that layer.